simplified-main/src/main/java/org/nypl/simplified/main/MainFragmentViewModel.kt

```diff
@@ -11,13 +29,150 @@ class MainFragmentViewModel : ViewModel() {
 
   private val logger = LoggerFactory.getLogger(MainFragmentViewModel::class.java)
 
-  /**
-   * `true` if the history of tabs should be cleared.
-   */
+  private val disposables = mutableListOf<Disposable>()
```

Comment:
There's actually a `CompositeDisposable` class in `rxjava` for this. Best to use that instead. It acts like a thread-safe mutable collection of `Disposable` values that is itself a `Disposable` value.